Description: revert usage of Crypt::OpenSSL::Guess
 neither packaged nor needed
Origin: vendor
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2018-07-31

--- a/Makefile.PL
+++ b/Makefile.PL
@@ -1,6 +1,5 @@
 use ExtUtils::MakeMaker;
 use Config;
-use Crypt::OpenSSL::Guess 0.11 qw(openssl_inc_paths openssl_lib_paths);
 use 5.006;
 
 my ($libdir, $incdir);
@@ -16,8 +15,8 @@
 WriteMakefile(
   'NAME'	    => 'Crypt::OpenSSL::Random',
   'VERSION_FROM'    => 'Random.pm',
-  'LIBS'            => $libdir ? [ "-L$libdir $libs" ] : [ openssl_lib_paths() . " $libs" ],
-  'INC'             => $incdir ? "-I$incdir" : openssl_inc_paths(),
+  'LIBS'            => $libdir ? [ "-L$libdir $libs" ] : [ $libs ],
+  'INC'             => $incdir ? "-I$incdir" : "",
   'AUTHOR'          => 'Ian Robertson',
    (eval { ExtUtils::MakeMaker->VERSION('6.52'); 1 } ?
     ('CONFIGURE_REQUIRES' =>
